config/jobs/kubernetes-sigs/container-object-storage-interface/container-object-storage-interface.yaml

```diff
@@ -48,3 +62,109 @@ presubmits:
+        securityContext:
+          privileged: true
+
+  - name: pull-container-object-storage-interface-lint
+    cluster: eks-prow-build-cluster
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/container-object-storage-interface
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    annotations:
+      testgrid-dashboards: sig-storage-container-object-storage-interface
+      testgrid-tab-name: lint
+      description: Linters in container-object-storage-interface repo.
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250121-4aed057712-master
+        command:
+        - runner.sh
+        args:
+        - make
+        - lint
+        resources:
+          limits:
+            cpu: 2
+            memory: 4Gi
+          requests:
+            cpu: 2
+            memory: 4Gi
+        env:
+        - name: USE_DOCKERIZED_BUILD
+          value: "true"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+
+  - name: pull-container-object-storage-interface-check-generated-files
+    cluster: eks-prow-build-cluster
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/container-object-storage-interface
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    annotations:
+      testgrid-dashboards: sig-storage-container-object-storage-interface
+      testgrid-tab-name: check-generated-files
+      description: Check generated files in container-object-storage-interface repo.
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250121-4aed057712-master
+        command:
+        - runner.sh
+        args:
+        - ./hack/prow-check-generate.sh
+        resources:
+          limits:
+            cpu: 2
+            memory: 4Gi
+          requests:
+            cpu: 2
+            memory: 4Gi
+        env:
+        - name: USE_DOCKERIZED_BUILD
+          value: "true"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+
+  - name: pull-container-object-storage-interface-e2e
+    cluster: eks-prow-build-cluster
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/container-object-storage-interface
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    annotations:
+      testgrid-dashboards: sig-storage-container-object-storage-interface
+      testgrid-tab-name: e2e
+      description: E2e tests in container-object-storage-interface repo.
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250121-4aed057712-master
+        command:
+        - runner.sh
+        args:
+        - ./hack/prow-e2e.sh
+        resources:
+          limits:
+            cpu: 4
+            memory: 6Gi
+          requests:
+            cpu: 4
+            memory: 6Gi
```

Comment:
@shanduur I set slightly higher resource requests for e2e tests. Let me know if you think we should push this higher.

Reply:
I'd say that even 2c/4Gi should be enough. Let's leave that as is, I don't expect any issues.

Reply:
Much obliged! I'll open this for review now since the remaining questions are for the test infra reviewers.